context/app/static/js/components/detailPage/visualization/Visualization/Visualization.jsx

```diff
@@ -50,10 +50,14 @@ const visualizationStoreSelector = (state) => ({
   onCopyUrlSnackbarOpen: state.onCopyUrlSnackbarOpen,
   setOnCopyUrlSnackbarOpen: state.setOnCopyUrlSnackbarOpen,
 });
-
-function Visualization(props) {
-  const { vitData, uuid, hasNotebook } = props;
-
+const sharedInfoSnackbarProps = {
+  anchorOrigin: {
+    vertical: 'top',
+    horizontal: 'center',
+  },
+  autoHideDuration: 4000,
+};
```

Comment:
Move this down, just before the JSX, just to keep the definition and the usage closer together?

Reply:
I prefer to move any variable definitions which don't depend on the component's props/state outside of the function so that they aren't run each render. A negligible gain in the instance, but I want to be consistent.